rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/moderationrequest/Sw360ModerationRequestService.java

```diff
@@ -193,10 +242,114 @@ public Map<PaginationData, List<ModerationRequest>> getRequestsByState(User sw36
      */
     public ModerationState acceptRequest(ModerationRequest request, String moderatorComment, @NotNull User reviewer)
             throws TException {
+        ComponentService.Iface sw360ComponentClient = getThriftComponentClient();
+        ProjectService.Iface sw360ProjectClient = getThriftProjectClient();
+        LicenseService.Iface sw360LicenseClient = getThriftLicenseClient();
+        SPDXDocumentService.Iface sw360SPDXClient = getThriftSPDXDocumentClient();
+        DocumentCreationInformationService.Iface documentCreationInfoClient = getThriftDocumentCreationInfo();
+        PackageInformationService.Iface packageInfoClient = getThriftPackageInfo();
```

Comment:
Please move the client creation to respective case block to prevent unnecessary usage.

Reply:
Comment addressed.